modules/installation/con-resource-requirements-orchestrator.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies
+// assembly-install-rhdh-orchestrator-helm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="con-resource-requirements-orchestrator_{context}"]
+== Resource requirements for Orchestrator
+
+The Orchestrator flavor of {product} comes with the following default resource requests and limits as defined in the Helm chart:
+
+[cols="2,2", options="header"]
+|===
+| Resource
+| Default value
+
+| CPU requests
+| 250m
+
+| Memory requests
+| 64Mi
+
+| CPU limits
+| 500m
+
+| Memory limits
+| 1Gi
+|===
```

Comment:
I think users would only be interested in resource limits (not requests), as this is what is enforced by the cluster.